legal-api/report-templates/certificateOfAmalgamation.html

```diff
@@ -24,40 +24,18 @@
                     <div class="sub-header-2">{{ entityAct }}</div>
                 </div>
 
-                <div class="certify-clause">
+                <div class="certify-clause-container">
                     <div>I hereby certify that the following businesses were amalgamated under the name</div>
                     <div class="company-name-header">{{ business.legalName }}</div>
                     <div>on {{ effective_date_time }}</div>
-                    <div>&nbsp;</div>
+                    <div class="mt-2">
                     {% for entity in amalgamatingBusinesses %}
-                        {% if entity == amalgamatingBusinesses[-1] %}
-                        <tr class="no-page-break">
-                            <td class="col-50">
-                                <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                    {% if entity.legalName %}
-                                    and {{ entity.legalName }} ({{ entity.identifier }})
-                                    {% else %}
-                                    and {{ entity.name }} ({{ entity.identifier }})
-                                    {%endif%}
-                                </div>
-                            </td>   
-                            </tr>
-                        {%else%}
-                        <tr class="no-page-break">
-                        <td class="col-50">
-                            <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                {% if entity.legalName %}
-                                {{ entity.legalName }}
-                                ({{ entity.identifier }}),
-                                {%else%}
-                                {{ entity.name }}
-                                ({{ entity.identifier }}),
-                                {%endif%}
-                            </div>
-                        </td>
-                        </tr>
-                        {%endif%}
+                        {% if entity != amalgamatingBusinesses[0] %}
```

Comment:
would this leave us with multiple 'and's ? For ex. would this make amalgamatingbusiness1 **and** amalgamatingbusiness2 **and** amalgamatingbusiness3? I thought we need the 'and' only at the end.

Reply:
![image](https://github.com/bcgov/lear/assets/64869009/8bf0ba30-f28e-4331-a0e3-182f5828ed25)

Reply:
https://preview.uxpin.com/7f4cfe5fe4b0ba15a83d518177efe82a08e99167#/pages/160598905/comment/sitemap

Reply:
If that's how we're doing it, looks great 👍